### DISCUSSION TITLE
Counting T-shirt in a stack.

### DISCUSSION THREAD

**Discussion by sudheer-palaparambil** [ultralytics/yolov5#10403] [Ideas]
Hello,

Is it possible to count number of T-shirts in each stack from the below given image using Yolov5?  

![Screenshot 2022-12-05 at 8 44 07 AM](https://user-images.githubusercontent.com/9995485/205541693-c9cd4cb9-b21c-4842-b130-94c539dc1095.png)

is this 
![Capture2](https://user-images.githubusercontent.com/9995485/205549672-90866558-f128-49db-be30-8ebdcc2c13db.JPG)
or this
![Capture3](https://user-images.githubusercontent.com/9995485/205549716-df6f5c9e-058f-4a4f-830c-55d02dcaae91.JPG)
is the right way to annotate the image? Or is there any other way to annotate the object in image?

Please help.

Thank you

Sudheer

**Comment by glenn-jocher** [comment]
@sudheer-palaparambil good question. Maybe the tags could also be labelled instead of the shirts, but since not all shirts have tags I guess yes you need to label each shirt. 

Your task can be solved with detection or segmentation labels as you have, but it might also fall under crowd-counting style AI's that simply produce heatmaps and estimated counts. 

For best results I'd do the middle image, it's got much better annotation.

**Reply by psudheers** [reply]
Thank you for your prompt reply.
